packages/flutter/lib/src/material/tabs.dart

```diff
@@ -70,6 +69,7 @@ class Tab extends StatelessWidget implements PreferredSizeWidget{
     this.text,
     this.icon,
     this.iconMargin = const EdgeInsets.only(bottom: 10.0),
+    this.TextAndIconTabHeight = 72,
```

Comment:
Is this meant to only work when there is a an icon & text in the tab? What if the user wants a custom height with just Text? Or just an Icon?

Reply:
Parameters should also start with a lowercase letter.

Reply:
sorry
i've forgotten to change the "T" to "t" after removed the "k"